appium/webdriver/extensions/power.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium import webdriver
+from ..mobilecommand import MobileCommand as Command
+
+
+class Power(webdriver.Remote):
+
+    def set_power_capacity(self, percent):
```

Comment:
usually we add `return self` for chaining if a method does not return anything

Reply:
Thanks, didn't know that